src/password/scripts/directives.js

```diff
@@ -90,11 +112,13 @@
                         .then(function (response) {
                             scope.fields = response.data.actions.PUT;
                         }, function (response) {
-                            // If the response returns a 404 and we have a token.
-                            if (response.status === 404 && angular.isDefined(TOKEN)) {
-                                // There was a problem with the token.
+                            if (angular.isDefined(TOKEN)) {
                                 scope.tokenError = true;
                             }
+                            if (response.status !== 404 && angular.isDefined(response.data)) {
+                                var errors = catchErrors.all({}, response);
```

Comment:
I think you should be able to get all the errors without having to provide an empty object in place of fields.